code/modules/clothing/head/tinfoilhat.dm

```diff
@@ -67,3 +70,22 @@
 	. = ..()
 	if(!warped)
 		warp_up()
+
+/obj/item/clothing/head/foilhat/suicide_act(mob/living/user)
+	user.visible_message(span_suicide("[user] gets a crazed look in [user.p_their()] eyes! [capitalize(user.p_they())] [user.p_have()] witnessed the truth, and try to commit suicide!"))
+	var/conspiracy_line = pick(list(
```

Comment:
```suggestion
	var/static/conspiracy_line = pick(list(
```
Static list since it's just a bunch of lines

Reply:
pick()